proposals/nomination-schedule.md

```diff
@@ -0,0 +1,24 @@
+# Proposed Nomination Schedule
+
+
+
+| Event                          | Time Taken  | What's involved |
+| :------------------------------ |:-----------| :---------------|
+| Marketing of board nominations | 1 week      | - Raise awareness of the board nominations coming up. <br/> - Share requirements for being eligible for being a board member. <br/> - Share the commitments and responsibilities of being a board member.<br/> - Share how to nominate yourself or someone else for the board. <br/> - Share how and why to become a member of the .NET Foundation.  |
```

Comment:
From committee discussion: We should add time for forming the nominating committee. That responsibility doesn't fall entirely to the membership committee, but does take time. My notes say this would take 3 weeks.